src/keys/x25519.rs

```diff
@@ -16,6 +16,8 @@ use core::convert::TryInto;
 #[cfg(feature = "ed25519")]
 use crate::signatures::ed25519;
 
+use zeroize::{Zeroize, ZeroizeOnDrop};
```

Comment:
Sorry I'm nitpicking but can you put the import between core and crate imports please? A new section with newline, to be consistent with our usual format.